src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Security/RequestHeaderConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Security;
```

Comment:
Might be better to put this in a more generic namespace as we could use this class for other request headers in the future.

`Requests`, `Utils` or a new namespace like `Constants` would probably be acceptable places

Reply:
Good idea, it's now `RequestHeaderNames` in `Public.Data.Api.Constants`, thanks.